bite-size-notes/week-8-d-Probability and Recursive Bayesian Filtermd.md

```diff
@@ -0,0 +1,210 @@
+## Preliminaries
+First, we'll review the variables and space of probability and what each of them means.
+### Random variables
+$\mathbf{X}$: A quantity whose value is unknown/uncertain to us.
+
+$\mathbf{x}$: A possible value of $\mathbf{X}$, or sample of $\mathbf{X}$, or realization of $\mathbf{X}$.
+### Probability space (Ω，F，P)
+Ω: Sample space ( all possible outcomes of random variable ).
+
+F: Event space( all possible sets of outcomes in Ω ).
+
+P: Probability measure (assigns probability to each event ).
+### Basis of probability
+And back to the function：
+
+The characteristic of continuous space is that random variables can take continuous values. Assume that all continuous random variables in this text have a probability density function (PDF). The common density function is the one-dimensional normal distribution (normal distribution) with mean $\mu$ and variance $\sigma^2$. The probability density function of the normal distribution can be given by the Gaussian function:
+
+$$
+p(x) = (2\pi\sigma^2)^{-\frac{1}{2}} \exp \left( -\frac{1}{2} \frac{(x - \mu)^2}{\sigma^2} \right) \tag{1} 
+$$
+
+This can be abbreviated as $N(x; \mu, \sigma^2)$.
+
+Equation (1) assumes that $x$ is a scalar quantity, but $x$ is often a multi-dimensional vector. The normal distribution of vectors is called multivariate. The density function of the multivariate normal distribution is:
+
+$$
+p(x) = \det(2\pi\Sigma)^{-\frac{1}{2}} \exp \left( -\frac{1}{2} (x - \mu)^T \Sigma^{-1} (x - \mu) \right) \tag{2}
+$$
+
+
+where $\mu$ is the mean vector, $\Sigma$ is a positive semidefinite and symmetric matrix called the covariance matrix; $T$ denotes the transpose of the vector.
+
+The joint distribution of two random variables $\mathbf{X}$ and $\mathbf{Y}$ is:
+
+$$
+p(x, y) = p(X = x, Y = y) \tag{3}
+$$
+
+If $\mathbf{X}$ and $\mathbf{Y}$ are independent, we have:
+
+$$
+p(x, y) = p(x)p(y) \tag{4}
+$$
+
+Assume that the value of $\mathbf{Y}$ is known to be $y$, and we want to know the probability of $\mathbf{X}$ being $x$ given this fact, expressed as
+
+$$
+p(x|y) = p(X = x | Y = y) \tag{5} 
+$$
+
+This is called the conditional probability. If $p(y) > 0$, the conditional probability is defined as
+
+$$
+p(x|y) = \frac{p(x,y)}{p(y)} \tag{6} 
+$$
+
+If $\mathbf{X}$ and $\mathbf{Y}$ are independent, then
+
+$$
+p(x|y) = \frac{p(x)p(y)}{p(y)} = p(x) \tag{7} 
+$$
+
+In other words, if $\mathbf{X}$ and $\mathbf{Y}$ are independent, then $\mathbf{Y}$ provides no information about the value of $\mathbf{X}$.
+
+### Theorem of Total Probability
```

Comment:
Here as well, some context or explanation of the theorem and its meaning is useful

Reply:
Thank you, I have added a brief explanation.